src/nvim/syntax.c

```diff
@@ -64,7 +64,7 @@ struct hl_group {
   int sg_cterm_bold;            ///< bold attr was set for light color
   // for RGB UIs
   int sg_gui;                   ///< "gui=" highlighting attributes
-                                ///< (combination of \ref HL_ATTRIBUTES)
+                                ///< (combination of HlAttrFlags)
```

Comment:
Is the HlAttrFlags picked up  automatically maybe ?

Reply:
I don't know, guess I have to check the output.